pkgs/applications/audio/picard/default.nix

```diff
@@ -1,7 +1,12 @@
-{ stdenv, python3Packages, fetchFromGitHub, gettext, chromaprint, qt5 }:
+{ stdenv, python3Packages, fetchFromGitHub, gettext, chromaprint, qt5, enablePlayback ? false }:
```

Comment:
I'm not sure whether it should default to `true`..

Reply:
I'm for enable-by-default, but I don't see any playback features in the new build or the changelog.

Reply:
TBH, now I notice this feature doesn't seem to work yet, but it does exist:

![NUX-selection:2020 03 27-13:54:15](https://user-images.githubusercontent.com/10998835/77749871-b36dce80-701a-11ea-9247-5eaf4867b51e.png)

See the player at the bottom.

Also, launching `picard` with `--debug` gives:

```
...
D: 14:05:23,792 /nix/store/fgfg1gy5mwigm7a4h67vqv3plhax11q1-picard-2.3.1/lib/python3.7/site-packages/picard/ui/playertoolbar.__init__:91: Internal player: QtMultimedia available, initializing QMediaPlayer
D: 14:05:23,800 /nix/store/fgfg1gy5mwigm7a4h67vqv3plhax11q1-picard-2.3.1/lib/python3.7/site-packages/picard/ui/playertoolbar.__init__:98: Internal player: available, QMediaPlayer set up
...
```

I tested both latest version and `master` and I opened upstream bug report: https://tickets.metabrainz.org/browse/PICARD-1794 .

Reply:
That got fixed now thanks to missing gstreamer plugins I added as well.